src/app/window-utils.ts

```diff
@@ -211,6 +211,8 @@ export const createComponentWindow = (
     show: false,
     width: 300,
     ...opts,
+    roundedCorners: isWindowsOS ? true : false,
```

Comment:
Component windows are not affected right? Do we need modify this here?

Reply:
roundedCorner will apply to every frame. Because right now those frames are also sharped edge